src/main/java/org/threeten/extra/chrono/InternationalFixedChronology.java

```diff
@@ -0,0 +1,484 @@
+ *  This is the same rule in use for the Gregorian calendar.
+ * <li>Week day - every month starts on a Sunday.  Leap-day and year-day are considered month-less Sundays.
+ * </ul>
+ *
+ * <h3>Implementation Requirements</h3>
+ * This class is immutable and thread-safe.
+ */
+public final class InternationalFixedChronology extends AbstractChronology implements Serializable {
+    /**
+     * Singleton instance for the International fixed chronology.
+     */
+    public static final InternationalFixedChronology INSTANCE = new InternationalFixedChronology();
+    /**
+     * Serialization version UID.
+     */
+    private static final long serialVersionUID = -1050847936470712250L;
+    /**
+     * Standard 7-day week.
+     */
+    static final int DAYS_IN_WEEK = 7;
+    /**
+     * In all months, there are 4 complete weeks.
+     */
+    static final int WEEKS_IN_MONTH = 4;
+    /**
+     * There are 13 months in a year.
+     */
+    static final int MONTHS_IN_YEAR = 13;
+    /**
+     * There are 4 weeks of 7 days, or 28 total days in a month.
+     */
+    static final int DAYS_IN_MONTH = WEEKS_IN_MONTH * DAYS_IN_WEEK;
+    /**
+     * Range of day-of-week.
+     */
+    static final ValueRange DAY_OF_WEEK_RANGE = ValueRange.of(1, DAYS_IN_WEEK);
+    /**
+     * There are 13 months of 28 days, or 365 days in a (non-leap) year.
+     */
+    static final int DAYS_IN_YEAR = MONTHS_IN_YEAR * DAYS_IN_MONTH + 1;
+    /**
+     * There are 52 weeks in a year.
+     */
+    static final int WEEKS_IN_YEAR = DAYS_IN_YEAR / DAYS_IN_WEEK;
+    /**
+     * The number of days in a 400 year cycle.
+     */
+    private static final int DAYS_PER_CYCLE = 146097;
+    /**
+     * The number of days from year zero to year 1970.
+     * There are five 400 year cycles from year zero to 2000.
+     * There are 7 leap years from 1970 to 2000.
+     */
+    static final long DAYS_0000_TO_1970 = (DAYS_PER_CYCLE * 5L) - (30L * 365L + 7L);
+    /**
+     * Range of year.
+     */
+    static final ValueRange YEAR_RANGE = ValueRange.of(1, 999_999);
+    /**
+     * Epoch day range.
+     */
+    static final ValueRange EPOCH_DAY_RANGE = ValueRange.of(-1L * DAYS_0000_TO_1970, 1_000_000L * DAYS_IN_YEAR);
+    /**
+     * Range of proleptic month.
+     */
+    private static final ValueRange PROLEPTIC_MONTH_RANGE = ValueRange.of(13, 1_000_000 * 13L - 1);
+    /**
+     * Range of day of month.
+     */
+    static final ValueRange DAY_OF_MONTH_RANGE = ValueRange.of(1, DAYS_IN_MONTH);
+    /**
+     * Range of week of year.
+     */
+    static final ValueRange WEEK_OF_YEAR_RANGE = ValueRange.of(1, WEEKS_IN_YEAR);
+    /**
+     * Range of aligned week of month, the weeks are always perfectly aligned.
+     */
+    static final ValueRange WEEK_OF_MONTH_RANGE = ValueRange.of(1, WEEKS_IN_MONTH);
+    /**
+     * Range of day of year.
+     */
+    static final ValueRange DAY_OF_YEAR_NORMAL_RANGE = ValueRange.of(1, DAYS_IN_YEAR);
```

Comment:
You probably don't need to create this yourself, as you should be able to piggyback off the default one.